frontend/src/__tests__/cypress/cypress/fixtures/e2e/dataScienceProject.yaml

```diff
@@ -0,0 +1,4 @@
+# cypress/fixtures/e2e/dataScienceProject.yaml
```

Comment:
Can you set the test and data files for the E2E with more common file names, for example:

- ds_projects_test.cy.ts
- ds_projects_data.cy.json

Also note that JSON is more common than YAML for Cypress data files.

Reply:
@manosnoam can you review the file names for the Mock tests? They seem to use camel case and not underscores.

Reply:
ack, in that case let's use same naming conventions.
In Mock they are called for example: `mockAllowedUsers.ts`
We can use:
`testDSProjects.ts` to indicate its e2e tests.

And according to existing fixtures for the tests, maybe we should also keep the current structure:
`frontend/src/__tests__/cypress/cypress/fixtures/resources/yaml`
In there place the relevant data for DS projects tests as `testDSProjects.yaml`

?

Reply:
Would 'testProjectCreationDeletion' be more relevant? 

The folder structure allows us to use '...dataScienceProject.yaml' for all the test cases related to dataScienceProjects. We should then create similar files for workbenches etc. 
frontend/src/__tests__/cypress/cypress/fixtures/e2e/dataScienceProject.yaml

As we scale the tests over time, having this granularity becomes particularly useful.

Reply:
My recent commit has resolved this issue - test names were changed along with directories.